config/zones/CA-ON.yaml

```diff
@@ -107,4 +112,7 @@ flag_file_name: ca.png
 parsers:
   price: CA_ON.fetch_price
   production: CA_ON.fetch_production
+sources:
+  Life cycle greenhouse gas emissions of electricity generation in the province of Ontario Canada 2013:
+    link: https://doi.org/10.1007/s11367-012-0501-0
```

Comment:
Nice with a DOI link! 🎉

Reply:
I hate link rot! I wonder if this could become a new standard for emaps....

Reply:
We should definitely push for DOI links if possible. I'll see if I can find some DOI links for the other sources as well.